### PR TITLE
Expanded AWS Console examples

### PR DESCRIPTION
**Pull request by stevenGravy**
Show setting up example read only and power user role access.  Expanded setting and starting app agent.   

Minor correction to mongodb file names in instructions.

### CODE REVIEW

**Review by Valien — APPROVED** [review]

File: docs/pages/application-access/guides/aws-console.mdx [hunk 1]
```diff
@@ -20,8 +21,49 @@ This guide will explain how to:
 - Teleport with Application Access. Follow [Getting Started](../getting-started.mdx)
   or [Connecting Apps](./connecting-apps.mdx) guides to get it running.
 - IAM permissions in the AWS account you want to connect.
+- AWS EC2 or other instance where you can assign a IAM Security Role for the Teleport Agent
 
-## Step 1. Update IAM roles trust relationships
+
+## Step 1. Configure Read Only and Power User Roles
```
Comment (r0mant) [inline docs/pages/application-access/guides/aws-console.mdx:27]:
```suggestion
## Step 1. Configure Read Only and Power User roles
```

File: docs/pages/application-access/guides/aws-console.mdx [hunk 2]
```diff
@@ -12,6 +12,7 @@ appropriate IAM roles.
 This guide will explain how to:
 
 - Connect your AWS account(-s) to Teleport.
+- Setup example AWS IAM Read Only and Power User Roles
```
Comment (r0mant) [inline docs/pages/application-access/guides/aws-console.mdx:15]:
```suggestion
- Setup example AWS IAM Read Only and Power User roles.
```

File: docs/pages/application-access/guides/aws-console.mdx [hunk 3]
```diff
@@ -20,8 +21,49 @@ This guide will explain how to:
 - Teleport with Application Access. Follow [Getting Started](../getting-started.mdx)
   or [Connecting Apps](./connecting-apps.mdx) guides to get it running.
 - IAM permissions in the AWS account you want to connect.
+- AWS EC2 or other instance where you can assign a IAM Security Role for the Teleport Agent
```
Comment (r0mant) [inline docs/pages/application-access/guides/aws-console.mdx:24]:
```suggestion
- AWS EC2 or other instance where you can assign a IAM Security Role for the Teleport Agent.
```

File: docs/pages/application-access/guides/aws-console.mdx [hunk 4]
```diff
@@ -20,8 +21,49 @@ This guide will explain how to:
...
 
-## Step 1. Update IAM roles trust relationships
+
+## Step 1. Configure Read Only and Power User Roles
+
```
Comment (r0mant) [inline docs/pages/application-access/guides/aws-console.mdx:28]: Do you think a note would be useful that would say something like "if you already have IAM role you want your users to impersonate, you can skip this step"?

File: docs/pages/application-access/guides/aws-console.mdx [hunk 5]
```diff
@@ -20,8 +21,49 @@ This guide will explain how to:
...
+AWS provides the `ReadOnlyAccess` and `PowerUserAccess` IAM policies that can be incorporated into roles.
+
+<Admonition type="note">
+These policies may provide too much or not enough access for your intentions. Validate these meet your expectations if you plan on using them.
+</Admonition>
+
+### Create the example Read Only Roles
```
Comment (r0mant) [inline docs/pages/application-access/guides/aws-console.mdx:35]:
```suggestion
### Create the example Read Only role
```

File: docs/pages/application-access/guides/aws-console.mdx [hunk 6]
```diff
@@ -20,8 +21,49 @@ This guide will explain how to:
...
+
+### Select the ReadOnlyAccess policy
+
+Press Next and Select the `ReadOnlyAccess`.  Sorting the Policy Name table from Z-A will make it faster to select.
+
+![Select Role Step 2](../../../img/application-access/create-role-example-readonly-2.png)
+
+Press next through the tags
+
+### Confirm Roles
```
Comment (r0mant) [inline docs/pages/application-access/guides/aws-console.mdx:50]:
```suggestion
### Confirm role
```

File: docs/pages/application-access/guides/aws-console.mdx [hunk 7]
```diff
@@ -20,8 +21,49 @@ This guide will explain how to:
...
+
+### Repeat for Power User
+
+Follow the same steps and select `PowerUserAccess` IAM Policy to create a `ExamplePowerUser` role.
+
+
+## Step 2. Update IAM roles trust relationships
+
+<Admonition type="note">
+This step is only required if you are allowing access from another account.  The trust relationship will already exist for the same account
```
Comment (r0mant) [inline docs/pages/application-access/guides/aws-console.mdx:65]:
```suggestion
This step is only required if you are allowing access from another account.  The trust relationship will already exist for the same account.
```

File: docs/pages/application-access/guides/aws-console.mdx [hunk 8]
```diff
@@ -97,25 +141,51 @@ spec:
...
+  enabled: "no"
+proxy_service:
+  enabled: "no"
+
+```
+
+Start the application service
+
+
+```bash
```
Comment (r0mant) [inline docs/pages/application-access/guides/aws-console.mdx:187]:
```suggestion
```code
```

**Review by r0mant — APPROVED** [review]
Just a few stylistic nits, lgtm after you address them.

I think @klizhentas might want to review this too.

File: docs/pages/application-access/guides/aws-console.mdx (hunk 4 above)
Comment (stevenGravy) [reply docs/pages/application-access/guides/aws-console.mdx:28]: added comment

**Comment by stevenGravy** [comment]
@r0mant addressed comments.  @klizhentas please do final review if u like

**Pull request merged**